src/Util/Migration/MigrationHelper.php

```diff
@@ -37,7 +37,7 @@ public function createPackLanguageTable(): void
                 CONSTRAINT `fk.swag_language_pack_language_language`
                     FOREIGN KEY (`language_id`)
                     REFERENCES `language` (`id`)
-                    ON DELETE RESTRICT
+                    ON DELETE CASCADE
```

Comment:
can you add a new migration? changing this will only affect new installations

Reply:
Hi @shyim 
Thank you for your review. 
I have added migration and reverted changes in the helper class.